### DISCUSSION TITLE
Discussion for the biweekly meeting (2022/09/28)

### DISCUSSION THREAD

**Discussion by sczyh30** [opensergo/opensergo-specification#44] [General]
Discussion for the biweekly meeting of the OpenSergo community (2022/09/28)

---

本次 OpenSergo 双周会将于 2022/09/28（周三）进行，欢迎大家预留时间参与，也欢迎大家发起 topic 来分享。议程如下：

* 同步上一周期的进展，包括微服务治理 spec 建设、Control Plane && SDK 进展、社区生态接入等进展
* 社区分享与讨论

主题：OpenSergo 双周会（钉钉群会议）
时间：2022年9月28日 周三 19:30 - 20:30
入会口令：244 801 51010
电话呼入：+862281944261（中国大陆）+867388953916（中国大陆）
入会链接：dingtalk://dingtalkclient/page/videoConfFromCalendar?confId=65d3c04b-b69e-44ce-b310-35054cff1d57&calendarId=1881630426

上次社区双周会记录：https://github.com/opensergo/opensergo-specification/discussions/41

**Comment by yyuuttaaoo** [comment]
关于日志治理，会议上讨论内容汇总如下：

日志治理上从可实现上考虑需要一些标准化的组件或者数据字段作为抓手，如日志等级、OpenTelemetry协议、特定的服务等。

生产端：
- 基于标准化的日志等级降级
- 基于OpenTelemetry数据的全链路数据采集
- OpenSerge等核心组件的标准日志

消费端
- 采集降级策略：暂停/采样
- 延时处理策略：丢弃
- 采集配置与使用OpenSergo部署的模块自动化

另外今天在OpenSergo周会上介绍的iLogtail，相关资料可以在https://github.com/alibaba/ilogtail进行查阅，欢迎关注转发~
